gita/common.py

```diff
@@ -2,8 +2,9 @@
 
 
 def get_config_dir() -> str:
-    root = os.environ.get('XDG_CONFIG_HOME') or os.path.join(
-        os.path.expanduser('~'), '.config')
+    root = os.environ.get('GITA_PROJECT_HOME') or 
+           os.environ.get('XDG_CONFIG_HOME') or 
```

Comment:
have u tried this on your computer? this will cause syntax error

Reply:
I fixed this syntax error.